src/main/app/claims/models/claim.ts

```diff
@@ -92,6 +92,20 @@ export class Claim {
     return this.respondedAt.clone().add('33', 'days')
   }
 
+  get respondToDefendantPaidInFullDeadline (): Moment {
+    if (!this.respondedAt) {
+      return undefined
+    }
+    return this.respondedAt.clone().add('28', 'days')
```

Comment:
don't we use sophisticated algorithm to calculate 28 working days + first working day after this period?

Reply:
this is a legal matter of 28 days. Separate from 28days + first working day.
http://www.justice.gov.uk/courts/procedure-rules/civil/rules/part15#15.10
```
(3) If the claimant fails to respond under this rule within 28 days after service of the court’s notice on him the claim shall be stayed(GL).

```

Reply:
do we not need to add service to this? service being 5 days.
i.e. this method already there?
respondToResponseDeadline

Reply:
@geek-soft will clarify with a PO for this but id like him to ask this question.

"If we allow the claimant to respond between 28 and 33 days after the response that the defendant has paid in full, does that open us up to any legal liability from the defendant claiming they were given too much time?"

Reply:
Clarified with PO that to should be 33 days. Thanks @jacmurphy for helping clarify it.